clusters/nishir/overlays/tailnet/ks.yaml

```
@@ -1056,3 +1056,26 @@ spec:
   sourceRef:
     name: flux-system
     kind: GitRepository
+  name: apps-inference
+  namespace: flux-system
+spec:
```

YAMLlint (1.37.1)

[error] duplication of key "spec" in mapping

(key-duplicates)